internal/kafka/internal/migrations/20221208124000_rename_kafka_storage_size_to_max_data_retention_size.go

```diff
@@ -0,0 +1,32 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func renameKafkaStorageSizeColumn() *gormigrate.Migration {
+	const KafkaStorageSize string = "kafka_storage_size"
+	const MaxDataRetentionSize string = "max_data_retention_size"
+
+	type KafkaRequest struct {
+		KafkaStorageSize     string `json:"kafka_storage_size"`
+		MaxDataRetentionSize string `json:"max_data_retention_size"`
+	}
+
+	return &gormigrate.Migration{
+		ID: "20221208160000",
```

Comment:
Let's update this file name and the `ID` of the name to a more recent date to make sure it has a more recent timestamp than all the other migrations.
This will need to be repeated each time the migration timestamp becomes outdated.